guides/common/attributes-base.adoc

```diff
@@ -16,16 +16,26 @@
 
 // Repositories and subscriptions (used both in Katello and Satellite guides)
 :SatelliteSub: Red Hat Satellite Infrastructure Subscription
+// For RHEl 7
 :RepoRHEL7Server: rhel-7-server-rpms
 :RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
 :RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
+// For RHEL 8
+:RepoRHEL8Server: rhel-8-for-x86_64-baseos-rpms
+:RepoRHEL8ServerSoftwareCollections: rhel-8-for-x86_64-appstream-rpms
```

Comment:
AppStream is not the same thing as SoftwareCollections.

These should really be `RepoRHEL8Base` and `RepoRHEL8AppStream`

Reply:
Resolved.